main.js

```diff
@@ -0,0 +1,74 @@
+const choices = ["rock","paper","scissors"]
+let playerScore = 0;
+let computerScore = 0;
+
+function playerChoice (){
```

Comment:
Recommend renaming your functions, this one to something like `getPlayerChoice` instead as `playerChoice` implies a variable that is set as the choice. It's a super minor naming convention thing that should make sense with time but always feel free to ask what functions should be named as if confused.